src/Core/System/Snippet/Filter/TermFilter.php

```diff
@@ -27,11 +27,11 @@ public function filter(array $snippets, $requestFilterValue): array
         $result = [];
         foreach ($snippets as $setId => $set) {
             foreach ($set['snippets'] as $translationKey => $snippet) {
-                $term = sprintf('*%s*', $requestFilterValue);
-                $keyMatch = fnmatch($term, $snippet['translationKey'], \FNM_CASEFOLD);
-                $valueMatch = fnmatch($term, $snippet['value'], \FNM_CASEFOLD);
+                $term = sprintf('/%s/i', $requestFilterValue);
```

Comment:
can't we use `mb_strpos` 🤔. Using Regex for this feels wrong 😅

Reply:
I tried that, but had a lot of problems. Will try that again, add it here.